src/components/card/index.js

```diff
@@ -0,0 +1,46 @@
+import { avatar } from "../avatar";
+import "./card.css"
+/**
+ * @typedef { Object } CardProps
+ * @property { string } CardProps.nome
+ * @property { string } CardProps.cargo
+ * @property { string } CardProps.img 
+ * @property { boolean } CardProps.expanded 
+ * @param { CardProps } props
+ */
+
+export function card({ nome, cargo, img, expanded }) {
+
+  if(expanded){
+    return `
+    <div class="card-expanded">
+          <div class="container-expanded">
```

Comment:
It's works, but note that change is just classes: card, container and body.

You can think about change this values using variable like this:


``` javascript
export function card({ nome, cargo, img, expanded }) {
const classExpanded = expanded ? '-expanded': '' ;

return `
    <div class="card${classExpanded}"> ...
`
```

This approach will be very common in react components